### ISSUE TITLE
Imagenet bounding boxes

### ISSUE THREAD

**Issue by frthjf** [tensorflow/datasets#248]
I was wondering whether it would be possible to retrieve the Imagenet bounding boxes through tfds. They are used for preprocessing in the tensorflow/models official resnet [here](https://github.com/tensorflow/models/blob/dadc4a625d273f995f8297968859c252147029ee/official/resnet/imagenet_preprocessing.py#L250) but don't seem to be available in tfds.

**Comment by captain-pool** [comment]
@frthjf This looks like a feature request or enhancement
@cyfra can you assign this to me?

**Comment by frthjf** [comment]
Thanks @captain-pool. That would be great to have!

**Comment by lgeiger** [comment]
@frthjf I agree, that would be great to have. @captain-pool Did you find time to look into fixing this issue?
